konig-schemagen/src/test/java/io/konig/schemagen/gcp/GoogleCloudResourceGeneratorTest.java

```diff
@@ -14,18 +16,24 @@
 import io.konig.core.impl.MemoryGraph;
 import io.konig.core.impl.RdfUtil;
 import io.konig.core.util.IOUtil;
+import io.konig.gcp.datasource.GcpShapeConfig;
 import io.konig.shacl.ShapeManager;
 import io.konig.shacl.impl.MemoryShapeManager;
 import io.konig.shacl.io.ShapeLoader;
 
 public class GoogleCloudResourceGeneratorTest {
 	
+	@Before
+	public void setup() {
+		GcpShapeConfig.init();
+	}
 
-	@Test
+	@Ignore
+	//@Test

```

Comment:
As a best practice, please do not comment out "@Test" or "@Ignore".  Just replace one with the other.  Having commented out annotations clutters the code and complicates global searches for ignored test cases.